src/components/DuosHeader.js

```diff
@@ -130,107 +260,153 @@ class DuosHeader extends Component {
+                  ])
+                ]),
+                li({}, [
+                  h(Link, { id: 'link_help', className: 'navbar-duos-link', to: '/FAQs' }, [
+                    div({ className: 'navbar-duos-icon-help', style: navbarDuosIcon }),
+                    span({ style: navbarDuosText }, ['FAQs'])
+                  ])
+                ]),
+                contactUsButton, supportrequestModal
               ])
-          })
+            ])
+          ]),
+        ]),
+        //NOTE: old navbar style is heavily dependent on css styles with element specific styles
+        //Hard to make that navbar flexible with material-ui's syntax
+        //For now I will use material-ui's hidden element to selectively render the two different navbars
+        //I'll look into rewriting the large navbar on a later PR
+        h(Hidden, {lgUp: true}, [
+          div({style: {display: 'flex', flexDirection: 'row', justifyContent: 'space-between'}}, [
+            img({
+              style: duosLogoImage, src: '/images/duos_logo.svg',
+              alt: 'DUOS Logo',
+              onClick: (e) => this.goToLink('/home')
+            }),
+            h(IconButton, {size: 'small', onClick: (e) => this.toggleDrawer(true)}, [
+              h(MenuIcon, {
+                style: {color: 'white', fontSize: '6rem', flex: 1},
+                anchor: 'right',
+              }, [])
+            ]),
+            h(Drawer, {
+              anchor: 'right',
+              open: this.state.openDrawer,
+              PaperProps: {
+                className: classes.drawerPaper
+              },
+              style: {
+                // I have to give this a ridiculous z-index value to keep the google sign in widget or register link from showing up on top
+              },
+              onClose: (e) => this.toggleDrawer(false)
+            }, [
+              h(List, {}, [
+                //NOTE: create user component to show logged in status (as well as dropdown options)
+                h(BasicListItem, {isRendered: isLogged, applyPointer, targetLink: '/profile', label: 'Your Profile', goToLink: this.goToLink}),
+                h(BasicListItem, {isRendered: isAdmin, applyPointer, targetLink: '/admin_console', label: 'Admin Console', goToLink: this.goToLink}),
+                h(BasicListItem, {isRendered: isChairPerson, applyPointer, targetLink: this.state.dacChairPath, label: 'DAC Chair Console', goToLink: this.goToLink}),
+                h(BasicListItem, {isRendered: isMember, applyPointer, targetLink: '/member_console', label: 'DAC Member Console', goToLink: this.goToLink}),
+                h(BasicListItem, {isRendered: isResearcher, applyPointer, targetLink: '/researcher_console', label: 'Researcher Console', goToLink: this.goToLink}),
+                h(BasicListItem, {isRendered: isResearcher, applyPointer, taretLink: '/data_ownder_console', label: 'Data Owner Console', goToLink: this.goToLink}),
```

Comment:
This link doesn't seem to work for me. The non-condensed version does work, but when in the hamburger menu, it seems to be a no-op.

Reply:
Ah ... I get it now. This link shows up regardless of the user role you have. If you don't have the role, it doesn't go anywhere. This should only be rendered if you have the role.

Reply:
No it's probably because I spelled "targetLink" wrong